src/Lecture4.hs

```diff
@@ -81,6 +81,8 @@ errors. You can just ignore invalid rows.
 Exercises for Lecture 4 also contain tests and you can run them as usual.
 -}
 
+{-# LANGUAGE StrictData #-}
```

Comment:
I see that you tried to implement the optimal lazy and streaming solution! What were your numbers? 😉

Reply:
Aha, I have tried, but it didn't work.